lib/bolt/util.rb

```diff
@@ -85,6 +85,44 @@ def first_run?
         Bolt::Config.user_path && !File.exist?(first_runs_free)
       end
 
+      def find_file_in_mod(mod, module_file)
+        subdir, relative_path = module_file.split(File::SEPARATOR, 2)
+        case subdir
+        when 'scripts'
+          unless relative_path.nil?
+            mod.file(module_file) || mod.script(File.join(mod, relative_path))
+          else
+            mod.file(module_file)
+          end
+        when 'files'
+          unless relative_path.nil?
+            mod.file(module_file) || mod.file(File.join(mod, relative_path))
+          else
+            mod.file(module_file)
+          end
+        else
+          mod.file(module_file)
```

Comment:
```suggestion
          mod.file(module_file) || mod.script(module_file)
```

I think this needs to fallback to `scripts`?